src/admin/dashboard/components/Header.tsx

```
@@ -42,91 +43,91 @@ const Header = () => {
-                                            <span className="w-1.5 h-1.5 rounded-full bg-[#7047EB]"></span>
-                                        ) }
-                                    </a>
-                                ) ) }
-                            </div>
+                                        />
+                                    </div>
+                                    <span
+                                        className={ `${
+                                            item?.active ? 'text-[#7047EB]' : ''
+                                        }` }
+                                        dangerouslySetInnerHTML={ {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)